### ISSUE TITLE
Hyperdimension Neptunia Victory - Inconsistent UI and texture display

### ISSUE THREAD

**Issue by Drvx** [RPCS3/rpcs3#10455]
This game still has major problem with UI:
![1](https://user-images.githubusercontent.com/74725652/121928902-4e9ab300-cd49-11eb-910c-b6f7688b4ef9.png)
![2](https://user-images.githubusercontent.com/74725652/121928905-4f334980-cd49-11eb-9fc1-675bc1556382.png)
![3](https://user-images.githubusercontent.com/74725652/121928911-50647680-cd49-11eb-867e-c748cd83a1f3.png)
![4](https://user-images.githubusercontent.com/74725652/121928917-5195a380-cd49-11eb-842c-abfcb0c8bf89.png)
![5](https://user-images.githubusercontent.com/74725652/121928928-54909400-cd49-11eb-9cca-d1e085d50942.png)

## This is how it's supposed to look: 
![1](https://user-images.githubusercontent.com/74725652/121929071-7e49bb00-cd49-11eb-995a-29f6e1e368f2.png)
![2](https://user-images.githubusercontent.com/74725652/121929541-fa440300-cd49-11eb-89ae-f993a40720b8.png)
![3](https://user-images.githubusercontent.com/74725652/121929082-80137e80-cd49-11eb-9d52-4f2b925115f7.png)
![4](https://user-images.githubusercontent.com/74725652/121929559-ff08b700-cd49-11eb-8b58-d608327eabf8.png)
![5](https://user-images.githubusercontent.com/74725652/121929574-029c3e00-cd4a-11eb-82f5-036245fab73c.png)

Some textures go wild too occasionally:
![6](https://user-images.githubusercontent.com/74725652/121930001-8a824800-cd4a-11eb-8e47-9fc6cf39e34c.png)

Tested with both Nvidia and AMD GPUs.
rpcs 0.0.16-12395
Win 10

[BLUS31150_20210614184011_capture.zip](https://github.com/RPCS3/rpcs3/files/6649909/BLUS31150_20210614184011_capture.zip)
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/6649912/RPCS3.zip)

**Comment by Asinin3** [comment]
Still an issue?

**Comment by Drvx** [comment]
> Still an issue?

Yes! 
Retested with 0.0.22-13660, nothing's changed.

**Comment by temp-account-hello** [comment]
Addendum to my last post: I probably should have tested further. [Here is a video example of what is happening.](https://youtu.be/akIPVd2pM08) For a quick description, when entering battle, the UI breaks once again and character/enemy portraits are replaced by black boxes, but this time they remain broken even after exiting battle. UI is broken every time.
I've also found out that installing the game data results in the black square replacing the portrait frame every time, so perhaps something to do with load times as well.
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12367805/RPCS3.log.gz)

**Comment by Triticum0** [comment]
@temp-account-hello  Retest, I only tested the demo but I couldn't see any graphical issue when using Read Color Buffers and Write Color Buffers after https://github.com/RPCS3/rpcs3/pull/14907.

**Comment by Monsterman07** [comment]
Some issues are still present like shown here with some characters
![rpcs3_2024 08 26-19 53](https://github.com/user-attachments/assets/5725008e-b8ab-487d-801e-f9f8a621fee6)
![rpcs3_2024 08 26-19 53_1](https://github.com/user-attachments/assets/390737ab-bd1a-450c-afe1-83637f75b4e1)
Also there's texture warping when switching characters in the same sub-menu (doesn't happen in dungeons + goes away later) and black boxes on the edges of the screen appear when loading a dungeon (for the first time I think). OpenGL might help with consistency. Other than those issues. I haven't sawn anything else in the game with the latest version of RPCS3.
screenshots but game res is 720p
![rpcs3_2024 08 27-21 28_1](https://github.com/user-attachments/assets/bda150a0-6a63-4242-b7f9-d8e55566bb6f)
![rpcs3_2024 08 27-21 28_2](https://github.com/user-attachments/assets/524aece8-adbe-409f-8371-98c76afbd9cb)

**Comment by digant73** [comment]
@Drvx, @temp-account-hello, @Monsterman07, @Triticum0 is that still an issue?

**Comment by digant73** [comment]
please re-test on latest build. Hopefully some textures and reflections issues have been fixed by recent build 0.0.39-18790

**Comment by Drvx** [comment]
@digant73 yeah, even with write + read color buffers there are still some graphical glitches as shown in previous posts (version 0.0.40-19193).
